.github/workflows/release.yaml

```diff
@@ -0,0 +1,54 @@
+name: Publish to PyPI
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'New version X.X.X'
+        required: true
+        type: string
+      pypi_instance:
+        # PyPI has a separate instance which can be used for testing purposes.
+        description: 'PyPI instance for publishing'
+        required: true
+        default: 'PyPI'
+        type: choice
+        options:
+        - 'TestPyPI'
+        - 'PyPI'
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Fetch all branches and tags.
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+      - name: Set Nextstrain bot as git user
+        run: |
+          git config --global user.email "78992647+nextstrain-bot@users.noreply.github.com"
```

Comment:
A real email might be more useful here, e.g. hello@nextstrain.org would suffice as a generic team email.

Reply:
Done. I personally use the GitHub-provided email to prevent spam in my inbox, but I guess it's too late for that with hello@nextstrain.org.